products/cpdb/commitments/metadata.yml

```diff
@@ -0,0 +1,141 @@
+name: commitments
+display_name: Capital Projects Database (CPDB) - Commitments
+summary: ""
+description: >
+  Office of Management and Budget (OMB) publishes the Capital Commitment Plan three times a year as
+  a series of pdf files, generally in the months of January, April, and September as part of
+  the publication of the Preliminary, Executive, and Adopted Capital Budgets.
+
+  
+  To explore the data, please visit 
+  <a href="https://capitalplanning.nyc.gov/map#10/40.7128/-74.0807">Capital Planning Explorer</a>
+
+
+  For additional information, please visit 
+  <a href="https://www.ibo.nyc.ny.us/iboreports/IBOCBG.pdf">A Guide to The Capital Budget</a>
+  
+tags: [capital projects database, capital planning, cpdb, budget, omb]
+each_row_is_a: Capital Commitment
+
+destinations:
+  - id: socrata_commitments
+    type: socrata
+    four_four: "djxg-kcfi"
+    datasets:
+      - commitments
+
+package:
+  dataset_files:
+    - name: commitments
+      filename: cpdb_planned_commitments.csv
+      type: csv
+
+  attachments:
+    - cpdb_data_dictionary.xlsx
```

Comment:
curious if this pushes correctly. If so, we can strike this [todo](https://github.com/NYCPlanning/data-engineering/blob/7a35e53bd04f0c9eb2bf3f418004f8a8e2aca4dd/dcpy/connectors/socrata/publish.py#L326)

Reply:
(discussed, see PR comments below)